holesky/README.md

```diff
@@ -133,15 +142,19 @@ time=2024-03-22T19:35:30.063Z level=INFO source=/app/node/node.go:351 msg="Store
 time=2024-03-22T19:35:30.063Z level=DEBUG source=/app/node/node.go:353 msg="Exiting process batch" 
 ```
 
-#### Step 7 (optional): To bring the containers down, run the following command
+#### Step 8 (optional): To bring the containers down, run the following command
 
 ```
 docker compose down
 ```
 
-Opt-Out of EigenDA
+#### Opt-Out of EigenDA
```

Comment:
This is placed at the same level of the steps above. I wonder if it'll make sense to structure the content around "jobs to be done" by an operator? Opt-out is a parallel job as opt-in.

Reply:
Agreed would prefer this is a level 2 or level 3 header.

Reply:
Sounds good. I'll change to level 3 header for now, open to whichever formatting you guys like.